index.bs

```diff
@@ -696,6 +698,39 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Signal platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method is used to signal users whether they can log in to the [RP] with the <a>platform-attached</a> authenticator. 
+The method gives [RP] the ability to detect whether the client is capable of registering credentials with 
+<a>platform authenticators</a> while giving users control over whether they want to share this information. When the 
+method is invoked, the <a>client</a> will search for the existence of an <a>platform authenticator</a> capable of 
+registering credentials. Upon finding one, the <a>client</a> will prompt the user for whether the user is interested
+to register a credential with the authenticator. If so, the platform will redirect the user to a new tab to register 
+the credentials. 
+
+When this method is invoked, the user agent MUST execute the following algorithm: 
+
+1. Let |promise| be a [=a new Promise=] and return the |promise|.
+
+1. Let |isInterested| be a [=boolean=] value. 
+
+1. For all |authenticators| currently available on this platform, if there exists an |authenticator| with 
+    the value of "platform" for its attachment modality, the <a>user agent</a> may take this opportunity to 
+    ask the user whether the user is interested in registering a credential with a <a>platform authenticator</a>.
+    The <a>user agent</a> should show three optional actions to the user: 
```

Comment:
FWIW, I like the idea of returning a `Promise` that resolves to indicate the presence of an authenticator or hangs to indicate non-presence. This could help deal with the case where the user attaches an authenticator for the first time after a website calls this function. Without a hanging `Promise`, we'd likely want another way to notify the website that the state has changed. With a hanging `Promise`, the UA can just resolve it discovers that the user has an authenticator. Hanging also avoids the potential information leak from the time it takes to return false.